base/code/src/components/Navbar.test.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { BrowserRouter } from 'react-router-dom';
+
+import { Navbar } from './Navbar';
+
+it('renders without crashing', () => {
+  const div = document.createElement('div');
```

Comment:
Wouldn't something like `const footer = create(<Footer />);` also work instead of rendering into the DOM?